common/protob/messages-common.proto

```diff
@@ -110,31 +111,16 @@ message PinMatrixAck {
  * @next PassphraseAck
  */
 message PassphraseRequest {
-    optional bool on_device = 1;    // passphrase is being entered on the device
+    optional bool _deprecated_on_device = 1;  // moved to PassphraseAck but left for backwards compatibility
```

Comment:
didn't we decide in the end that this will be left out?

Reply:
I wasn't sure to be honest. I wanted to double-check with you

Reply:
I originally wanted to keep this for backwards compatibility, BUT removing `PassphraseState` messages breaks that irreparably anyway
so if we wanted backwards compatibility that bad, we would need to keep these too -- which we don't want, I think?

Reply:
Oh, yeah! Yes, I think it's time to make break it. I will remove this.